docs/reference/search/request/preference.asciidoc

```diff
@@ -33,6 +33,10 @@ The `preference` is a query string parameter which can be set to:
 	and `3` in this case). This preference can be combined with other
 	preferences but it has to appear first: `_shards:2,3;_primary`
 
+`_only_nodes_spec`::
```

Comment:
yeah we can use that , but this one is actually entire node spec like we use in _cluster api  . I just copied _spec part from https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster.html

Let me know what you think . I can change to attributes if you guys think thats the right naming